apps/web-giddh/src/app/tallysync/completed/completed.component.scss

```diff
@@ -62,18 +62,23 @@
 }
 
 .form {
-    .company {
-        width: 205px;
-        padding-right: 0px;
-    }
+    // .company {
```

Comment:
Consider removing commented code if not required

Reply:
Done